### PR TITLE
Feature/spring boot

### PR DESCRIPTION
**Pull request by stephencattaneo**
ref: DEV-1473

### CODE REVIEW

File: spring-boot/backend/gradle/libs.versions.toml [hunk 1]
```diff
@@ -0,0 +1 @@
+../../../../../../gradle/libs.versions.toml
```
Comment (MeredithSchmidt) [inline spring-boot/backend/gradle/libs.versions.toml]: Where does this go...?

File: spring-boot/frontend/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev --experimental-https --experimental-https-cert=/code/certificates/localhost.pem --experimental-https-key=/code/certificates/localhost-key.pem",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@userfront/next": "^1.0.0",
```
Comment (isanchez-userfront) [inline spring-boot/frontend/package.json:12]: If you're only using `@userfront/next/client` you can drop-in `@userfront/react` instead.

File: spring-boot/backend/src/test/java/example/web/HelloControllerTests.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+import org.junit.jupiter.api.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class HelloControllerTests {
+
+	@Test
+	void myBadTest() throws Exception {
+		assertThat("asdf").isNotNull();
```
Comment (MeredithSchmidt) [inline spring-boot/backend/src/test/java/example/web/HelloControllerTests.java]: Going to keep this test?

Comment (stephencattaneo) [reply spring-boot/backend/src/test/java/example/web/HelloControllerTests.java]: ah. I'll figure out how to exclude tests in the build. I think its
```
test {
    exclude '**/*'
}
```

File: spring-boot/frontend/package.json (hunk 2 above)
Comment (stephencattaneo) [reply spring-boot/frontend/package.json:12]: I'm going to leave it as userfront/next to lower confusion. I think its fine for developers to keep the mindset of "I'm using nextjs, I should import `@userfront/next`".

File: spring-boot/backend/gradle/libs.versions.toml (hunk 1 above)
Comment (stephencattaneo) [reply spring-boot/backend/gradle/libs.versions.toml]: i'll delete that

**Review by isanchez-userfront — APPROVED** [review]

**Pull request merged**